check-code-style/check_style_bzl.sh

```diff
@@ -45,7 +45,7 @@ do
         tput bold # Bold text in terminal.
         printf "Command to format ${file}: "
         tput setaf 2 # Green font in terminal.
-        printf "buildifier "${file}"\n"
+        printf "buildifier --lint=fix --warnings=all "${file}"\n"
```

Comment:
I think we want `--lint=warn` here; it doesn't really make sense for this action to try to fix our code.

Reply:
This is not the command being run by the action (that's on line 37). Rather, this is the output to the user when the check has failed, to say "if you want to format the file in question correctly, run this command: `...`". 

So I think this line should in fact try to fix the code 😉

Reply:
🤦🏻 yes thank you. 😃